### ISSUE TITLE
Equivalence feature failures in acceptance tests

### ISSUE THREAD

**Issue by steven-sheehy** [hashgraph/hedera-mirror-node#7641]
### Description

#7342 seems to have caused a regression when running acceptance tests:

```
AcceptanceTest > Cucumber > EstimateGas Contract Base Coverage Feature > com.hedera.mirror.test.e2e.acceptance.AcceptanceTest.Validate EstimateGas with precompile FAILED
    io.cucumber.core.runner.DuplicateStepDefinitionException: Duplicate step definitions in com.hedera.mirror.test.e2e.acceptance.steps.HistoricalFeature.verifyMirrorAPIResponse(int) and com.hedera.mirror.test.e2e.acceptance.steps.EquivalenceFeature.verifyMirrorAPIResponse(int)
        at app//io.cucumber.core.runner.CachingGlue.lambda$prepareGlue$3(CachingGlue.java:278)
        at java.base@21.0.1/java.util.ArrayList.forEach(ArrayList.java:1596)
        at app//io.cucumber.core.runner.CachingGlue.prepareGlue(CachingGlue.java:272)
        at app//io.cucumber.core.runner.Runner.runPickle(Runner.java:72)
        at app//io.cucumber.junit.platform.engine.CucumberEngineExecutionContext.lambda$runTestCase$4(CucumberEngineExecutionContext.java:112)
        at app//io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:137)
        at app//io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
        at app//io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:137)
        at app//io.cucumber.junit.platform.engine.CucumberEngineExecutionContext.runTestCase(CucumberEngineExecutionContext.java:109)
        at app//io.cucumber.junit.platform.engine.NodeDescriptor$PickleDescriptor.execute(NodeDescriptor.java:168)
        at app//io.cucumber.junit.platform.engine.NodeDescriptor$PickleDescriptor.execute(NodeDescriptor.java:90)
        at java.base@21.0.1/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
        at java.base@21.0.1/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387)
        at java.base@21.0.1/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312)
        at java.base@21.0.1/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843)
        at java.base@21.0.1/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808)
        at java.base@21.0.1/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188)
```

Even after fixing that there are other random errors in equivalence:

```
    2024-01-29T22:50:12.256Z ERROR ForkJoinPool-1-worker-6 c.h.m.t.e.a.s.EquivalenceFeature Scenario failure: 
    I successfully create selfdestruct contract
    the mirror node REST API should return status 200 for the contracts creation
    I execute balance opcode against a contract with balance <- Illegal embedded sign character
    I execute selfdestruct and set beneficiary to the deleted contract address
     java.lang.NumberFormatException: Illegal embedded sign character
        at java.base/java.math.BigInteger.<init>(BigInteger.java:499)
        at java.base/java.math.BigInteger.<init>(BigInteger.java:679)
        at com.hedera.mirror.test.e2e.acceptance.response.ContractCallResponse.getResultAsNumber(ContractCallResponse.java:40)
        at com.hedera.mirror.test.e2e.acceptance.steps.EquivalenceFeature.balanceOfContract(EquivalenceFeature.java:117)
        at ✽.I execute balance opcode against a contract with balance(classpath:features/contract/equivalence.feature:26)

    2024-01-29T22:50:12.257Z  INFO ForkJoinPool-1-worker-10 c.h.m.t.e.a.s.HistoricalFeature Executing step: I create fungible and non-fungible token 
    2024-01-29T22:50:12.257Z  INFO ForkJoinPool-1-worker-6 c.h.m.t.e.a.s.EquivalenceFeature Executing step: I execute selfdestruct and set beneficiary to the deleted contract address 

AcceptanceTest > Cucumber > in-equivalence tests > com.hedera.mirror.test.e2e.acceptance.AcceptanceTest.Validate in-equivalence for selfdestruct and balance against contract FAILED
    java.lang.NumberFormatException: Illegal embedded sign character
        at java.base/java.math.BigInteger.<init>(BigInteger.java:499)
        at java.base/java.math.BigInteger.<init>(BigInteger.java:679)
        at com.hedera.mirror.test.e2e.acceptance.response.ContractCallResponse.getResultAsNumber(ContractCallResponse.java:40)
        at com.hedera.mirror.test.e2e.acceptance.steps.EquivalenceFeature.balanceOfContract(EquivalenceFeature.java:117)
        at ✽.I execute balance opcode against a contract with balance(classpath:features/contract/equivalence.feature:26)
```

### Steps to reproduce

`./gradlew :test:acceptance --info -Dcucumber.filter.tags=@acceptance`

### Hedera network

other

### Version

main

### Operating system

None

**Comment by jascks** [comment]
Note that this will be fixed when my PR #7565 is merged into main.

**Comment by steven-sheehy** [comment]
To clarify, the duplicate steps will be fixed in #7565 but there are other failures in equivalence feature that need to be addressed. Hence why we're disabling it for now.
